src/tileLayer.js

```diff
@@ -1358,6 +1366,17 @@ var tileLayer = function (arg) {
           m_this._purge(zoom, true);
         }
       );
+    // for tiles that aren't in view, remove them from the list of tiles that
+    // are needed to be loaded to be considered idle.
+    if (m_this._options.idleAfter !== 'all') {
+      for (const hash in m_promisedTiles) {
+        const tile = m_promisedTiles[hash];
+        if (tiles.indexOf(tile) < 0) {
```

Comment:
So is that the case that all the tiles that are not in view have index < 0, just wanted to make sure I understand the logic here.

Reply:
Yes.  If a value is not in an array, indexOf returns -1.  The tiles array is the list of tiles in the view.

Reply:
Okay thanks.